package/MDAnalysis/core/selection.py

```diff
@@ -573,6 +573,20 @@ class AltlocSelection(StringSelection):
     field = 'altLocs'
 
 
+class AromaticSelection(Selection):
+    """Contains all aromatic atoms"""
```

Comment:
I realise it's self-evident, but you might want to expand on the docstring here a little bit, at the very least to say which attribute you are selecting on. That way it's more obvious to users that are reading the documentation.